### PR TITLE
Added support to install istio-v1.5 and delete using istio-demo-auth

### PR DESCRIPTION
**Pull request by nveenjain**
This commit is due to revert done in
https://github.com/layer5io/meshery-istio/pull/62

This commit also resolves all CI error messages, including static and
runtime error checks

Signed-off-by: Naveen Jain <nveenjain@gmail.com>

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -21,7 +21,7 @@ jobs:
       uses: actions/setup-go@v1
       with:
         go-version: ${{ secrets.GO_VERSION }}
-    - run: GOPROXY=direct GOSUMDB=off go get -u golang.org/x/lint/golint; go list ./... | grep -v /vendor/ | xargs -L1 /home/runner/go/bin/golint -set_exit_status
+    - run: GOPROXY=direct GOSUMDB=off go get -u golang.org/x/lint/golint; go list ./istio/... | grep -v /vendor/ | xargs -L1 /home/runner/go/bin/golint -set_exit_status
```
Comment (nveenjain) [inline .github/workflows/ci.yml]: I've only allowed these tests to run on valild files that are edited and not auto generated, no error will be generated for auto-generated files, as expected!

**Comment by nveenjain** [comment]
All tests passed!

File: .github/workflows/ci.yml (hunk 1 above)
Comment (leecalcote) [reply .github/workflows/ci.yml]: @gunjanmimo @ashiscs have a look here and take note, if you would. This configuration could be propagated across other repos.

**Comment by ashiscs** [comment]
Got it Sir @leecalcote

**Comment by leecalcote** [comment]
> Got it Sir @leecalcote

Thank you, @ashiscs!

**Pull request merged**